examples/transformers/README.md

```diff
@@ -0,0 +1,189 @@
+# Transformers-SparseML Integration
+This folder contains an example on how to use sparseml with transformers. 
+We focus on Question answering and use a modified implementation from the BERT SQuAD in transformers. 
+Using various pruning configuration files we demostrate the effect unstructured pruning can have on SQuAD.
+
+## Installation and Requirements
+These example scripts require sparseml, transformers, torch, datasets and associated to libraries. To install run the following command
+
+```bash
+conda create -n sparsemlBERT python=3.7
```

Comment:
For the rest of the system, we've been going through instructions with virtualenv's and pip, let's migrate these instructions to that to standardize. No preference on conda over virtualenv for dev's local environments, but need to be consistent in docs